application/F3DOptionsParser.cxx

```diff
@@ -413,11 +409,18 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
 
       auto unmatched = result.unmatched();
 
-      if (unmatched.size() > 0)
+      inputs.clear();
```

Comment:
This `clear()` feels out of place but it's needed because `ConfigurationOptions::GetOptions` is called multiple times with `parseCommandLine=true` and I don't have a good enough understanding of the overall life-cycle of the app to come up with a better solution.

Reply:
indeed. I dont have a better solution to suggest, but adding a comment would go a long way